components/Stats.js

```diff
@@ -1,7 +1,16 @@
 import React from "react";
 
 const Stats = () => {
-	return <div>Stats</div>;
+	return (
+		<div>
+			<ul className='font-lexend text-5xl decoration-black font-semibold absolute left-6 bottom-11 leading-15 flex items-center text-center space-x-32'>
```

Comment:
<img width="1432" alt="image" src="https://user-images.githubusercontent.com/36285610/184818406-69063191-a9a7-4503-adc5-00571a4881a2.png">

The `absolute` is dangerous to use cause of the unexpected behavior it has in different situations such as above. Removing it allows it to be more responsive and less strict. Removing it gives you the expected behavior.